CoEpi/ui/RootWireframe.swift

```diff
@@ -61,6 +61,17 @@ class RootWireFrame {
         case .quiz: showQuiz()
         case .debug: showDebug()
         case .alerts: showAlerts()
+        case .thankYou: showThankYou()
+        case .breathless: showBreathless()
+        case .cough1: showCough1()
+        case .cough2: showCough2()
+        case .cough3: showCough3()
+        case .fever1: showFever1()
+        case .fever2: showFever2()
+        case .fever3: showFever3()
+        case .fever3Other: showFever3Other()
+        case .fever4: showFever4()
+        case .symptomReport: showSymptomReport()
```

Comment:
Is this all working? How is the back navigation?

Reply:
yes